app/controllers/votes_controller.rb

```diff
@@ -1,8 +1,15 @@
 class VotesController < ApplicationController
-  def create
-    topic = Topic.find(params[:topic_id])
-    vote = topic.votes.build
-    vote.save!
-    redirect_to(topics_path)
-  end
+  # def create
+  #   topic = Topic.find(params[:topic_id])
+  #   vote = topic.votes.build
+  #   vote.save!
+  #   redirect_to(topics_path)
+  # end
+
+  # def update
```

Comment:
For downvoting, I think the right thing to do is create a `def destroy` method:

The tricky thing is that usually the URL generated will be something like:

`DELETE /votes/:vote_id`

Since we don't have a specific Vote ID we want to destroy here (we only want to destroy `votes.last`), it gets a little weirder.

I think in this case, it might make more sense to nest the `VotesController` under `/topics`. You can do that like this:

`config/routes.rb`:
```ruby
resources :topics do
  resources :votes, only: [:create, :destroy] do
  end
end
```

This will generate URLs like `POST /topics/:topic_id/votes`, 'DELETE /topics/:topic_id/votes`

If you want to ever see a list of routes that you can easily look through, run:

`rake routes` in the root of your project via terminal

If the list gets too long, pipe it to `less` to be able to scroll with the arrow keys (`q` quits):

`rake routes | less`